mite_browser/__init__.py

```diff
@@ -88,7 +88,7 @@ async def request(self, method, url, *args, **kwargs):
         """Perform a request and return a page object"""
         # Wrap everything in page object
         embedded_res = kwargs.pop("embedded_res", self._embedded_res)
-        resp = await self._session.request(method, url, *args, **kwargs)
+        resp = await self._session.getattr("request")(method, url, *args, **kwargs)
```

Comment:
Like in my comment on the other PR, if this is necessary then it seems like the wrapper is not quite constructed right...